src/main/java/com/cosmian/jna/findex/structs/ChainTableValue.java

```diff
@@ -0,0 +1,19 @@
+package com.cosmian.jna.findex.structs;
+
+import com.cosmian.jna.findex.serde.Leb128ByteArray;
+
+/**
+ * An chain table value holds the encrypted content of an {@link IndexedValue} in the Chain table
+ */
+public class ChainTableValue extends Leb128ByteArray {
+    static final long serialVersionUID = 1L;
```

Comment:
What is this for? Didn't see any mention of it in other place in the code. Maybe add a comment if it's useful?

Reply:
Not useful anymore since it does not Implement Serializable anymore